From e2e7fb4d73d4a9bac28bd1d7319fa5f093f82f34 Mon Sep 17 00:00:00 2001
From: Olly Betts <olly@survex.com>
Date: Thu, 29 Apr 2010 12:13:31 +0000
Subject: [PATCH] configure.ac: Install headers under PREFIX/include not
 PREFIX/include/xapian.  This broke compiling code which didn't use
 XO_LIB_XAPIAN or xapian-config.

git-svn-id: file:///home/xapian-svn/xapian/trunk@14459 6b2f1b92-a4f3-0310-943f-de3cbe41748b
---
 xapian-core/configure.ac |    4 ++--
 2 files changed, 8 insertions(+), 2 deletions(-)

diff --git a/xapian-core/configure.ac b/xapian-core/configure.ac
index af8e2b6..9715a91 100644
--- a/xapian-core/configure.ac
+++ b/xapian-core/configure.ac
@@ -41,8 +41,8 @@ dnl Disabled for stable release series.
 dnl Default to "-1.1"; for no suffix, specify: --program-suffix=
 dnl test x"$program_suffix" != xNONE || program_suffix=-1.1
 
-dnl Where xapian.h, etc go.
-incdir=$includedir/xapian
+dnl Where xapian.h, etc go.  In development release append "/xapian-1.1".
+incdir=$includedir
 AC_SUBST(incdir)
 
 dnl Check the build directory doesn't contain a space, so we die early with
-- 
1.7.1

